conf/notify.conf

```diff
@@ -4,3 +4,7 @@ email_store = /srv/BOSS/emails
 default_sender = boss_no_reply@build.pub.meego.com
 # same as getbuildlog "logdir"
 allowed_attachment_dirs = /tmp
+# list of domain names that is allowed to receive emails
```

Comment:
This should say "space separated" since it's processed by .split()
It's too easy to assume commas for email type things
It should also say "If whitelist is given then all other domains are blacklisted by default"